purchase_work_acceptance/models/account_invoice.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    is_required_wa_id = fields.Boolean(
```

Comment:
As boolean, I think no need "_id" nor "is_"
```suggestion
    require_wa = fields.Boolean(
```